src/Services/Basket/Basket.API/Dockerfile

```diff
@@ -1,6 +1,7 @@
 #See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
 FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+RUN apt-get update && apt-get install -y curl
```

Comment:
Não tenho certeza que isso seja valido. A imagem ja é criada com suas versões corretas e definidas. Atualizar aqui imagino que só incrementa o tempo de geração e não traga nenhum benficio real.

Reply:
O comando `apt-get update` atualiza as informações sobre os repositorios dos pacotes, ele não atualiza nenhum pacote.
A imagem padrão do asp.net não vem com o wget ou com o curl, por isso `apt-get install -y curl` é utilizado, o healthcheck no Docker usa esse comando.

[Docker Healthchecks in ASP.NET](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-8.0#docker-healthcheck)